libfuse/fs.go

```diff
@@ -354,6 +354,12 @@ func (f *FS) Statfs(ctx context.Context, req *fuse.StatfsRequest, resp *fuse.Sta
 		Namelen: ^uint32(0),
 		Frsize:  fuseBlockSize,
 	}
+	if _, err := libkbfs.GetCurrentSessionIfPossible(
+		ctx, f.config.KBPKI(), false); err != nil {
```

Comment:
Ah I remember now. We are passing in `false` here. So it's this case: https://github.com/keybase/kbfs/blob/ad3b9231b631c41a62729cf6a10520e61c8829d0/libkbfs/kbpki_client.go#L223